yt/frontends/athena/data_structures.py

```diff
@@ -486,6 +488,7 @@ def __init__(
         self.specified_parameters = parameters.copy()
         if units_override is None:
             units_override = {}
+        self._magnetic_factor = cgs_normalizations[magnetic_normalization]
```

Comment:
One more idea to make this even better: instead of looking up the value directly from the dictionary, we could wrap it in a small helper function such as
```python
def get_magnetic_normalization(key:str) -> float:
    if not key in cgs_normalizations:
        raise ValueError(
            "Unknown magnetic normalization convention. "
            f"Got {key!r}, expected one of {tuple(cgs_normalizations)}
        )
    return cgs_normalizations[key]
```
so that valid values are visible in the error message (instead of the raw `KeyError`)